File: PHP/Program/SivitasAkademik.php
<?php

class SivitasAkademik extends Human
{
    private $asalUni = "";
    private $email = "";

    public function __construct($asalUni = "", $email = ""){
        $this->asalUni = $asalUni;
        $this->email = $email;
    }

    // Get-set Asal Universitas
    public function setAsalUni($asalUni){
        $this->asalUni = $asalUni;
    }

    public function getAsalUni(){
        return $this->asalUni;
    }

    // Get-set Email
    public function setEmail($email){
        $this->email = $email;
    }

    public function getEmail(){
        return $this->email;
    }
}

?>

File: Python/Program/SivitasAkademik.py
from Human import Human

class SivitasAkademik(Human):
    __asalUni = ""
    __email = ""

    def __init__(self, asalUni = "", email = ""):
        self.__asalUni = asalUni
        self.__email = email
    
    # Get-set Asal Universitas
    def setAsalUni(self, asalUni):
        self.__asalUni = asalUni

    def getAsalUni(self):
        return self.__asalUni

    # Get-set Email
    def setEmail(self, email):
        self.__email = email

    def getEmail(self):
        return self.__email